### DISCUSSION TITLE
file specified with --config flag not shipping w/ zip file for serverless

### DISCUSSION THREAD

**Discussion by bobpozun** [artilleryio/artillery#1629] [Artillery]
starting to use the aws lambda functionality and i've noticed that the globals file isn't shipping in the zip file [edit: globals file is added via the --config parameter]. does that sound like an issue or is there something i need to do to have it included?

**Comment by hassy** [comment]
hi @bobpozun 👋 not sure what you mean. Which globals file are you referring to?

**Reply by bobpozun** [reply]
hi! ohhh sorry, I forgot to mention that for my project I have a globals.yml file that I pass in via the --config flag which contains shared config to reduce duplicated lines across all of my scripts. I'll update the discussion title..

**Reply by bobpozun** [reply]
you can disregard this --- i mistakenly didn't see this as a current limitation in the docs

**Reply by hassy** [reply]
No worries :) that'll be implemented in the upcoming release!

**Reply by hassy** [reply]
This will be supported in the next release, landing in #1650

**Reply by hassy** [reply]
@bobpozun this should work in v2.0.0-25 now!
